aws-lambda-java-events-sdk-transformer/pom.xml

```diff
@@ -85,6 +85,28 @@
         <artifactId>maven-failsafe-plugin</artifactId>
         <version>2.22.2</version>
       </plugin>
+      <plugin>
+        <groupId>org.moditect</groupId>
+        <artifactId>moditect-maven-plugin</artifactId>
+        <version>1.0.0.Final</version>
```

Comment:
Latest version seems to be 1.2.2, could you please update it?
https://mvnrepository.com/artifact/org.moditect/moditect-maven-plugin/1.2.2.Final